README.md

```diff
@@ -121,6 +121,11 @@ The following options are also available:
   Default: false
 * impact - analyze local changes to determine which hosts/roles to test.
   Default: false
+* ssh_cmd_gen_template - A variable to supply a custom ssh command
+  generator which will be invoked by TT to generate a vanilla command.
+  This can be expressed in a template format, with the arguments
+  supplied using `ssh_cmd_gen_args`
+  Default: nil
```

Comment:
Jeez, you really want them to have to read the source, don't you? :)

Maybe something more like...

```suggestion
* ssh_cmd_gen_template - A command to run, whose stdout will be the
  ssh command for  to supply a custom ssh command Taste Tester will
  run to access test nodes. The command should be a template and the
  will be passed, at minimum, the following variables:
    * `jumps` - any jumphost arguments
    * `host` - the host to SSH to
    * `user` - the user to SSH as
  For example:
    `ssh_gen %{jumps} --user %{user} %{host}`
  In addition, further arguments can be password using `ssh_cmd_gen_args`
  Default: nil
```